src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.Cache.cs

```diff
@@ -243,140 +237,6 @@ internal JsonPropertyInfo CreatePropertyUsingReflection(Type propertyType, Type?
             return info;
         }
 
-        // AggressiveInlining used although a large method it is only called from one location and is on a hot path.
```

Comment:
This logic is being deleted because of the newly added `JsonPropertyInfo.ParameterInfo` property, which removes the need for a separate index for parameters.

Reply:
Is it possible that this regresses performance in JsonConstructor cases?

Reply:
I don't think so. If anything it should improve it because it now does one lookup instead of two.